js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1206,7 +1206,7 @@
 
     checkDatabasePermissions: function (roCallback, rwCallback) {
       var url = arangoHelper.databaseUrl('/_api/user/' +
-        encodeURIComponent(window.App.userCollection.activeUser) +
+        encodeURIComponent(window.App.userCollection.activeUser || "root") +
```

Comment:
How easy is it to add tests for such things?

Reply:
To implement unit tests for the complete current UI is kind of an impossible task as the code is kind of not prepared for that. There are some automated unit tests I have to have a look into. We could add a test for this read-only-scenario with "disabled authentication". But this would test the scenario and not the added lines of code.